server/container_create.go

```diff
@@ -904,6 +904,26 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		return nil, err
 	}
 
+	containerMounts := specgen.Spec().Mounts
+	// bind mount default user defined secrets
+	overrideMounts, err := secretMounts(mountLabel, overrideMountCfgFile, containerInfo.RunDir, containerMounts)
+	if err != nil {
+		return nil, err
+	}
+	for _, m := range overrideMounts {
+		specgen.AddBindMount(m.Source, m.Destination, nil)
+
+	}
+	containerMounts = specgen.Spec().Mounts
```

Comment:
this can be dropped I guess (?)

Reply:
I need an updated list of mounts after going through the override file to ensure that if the default files has the same mounts, it does not duplicate and override existing mounts.